services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -145,7 +148,15 @@ async def start_project_interactive_services(
         )
         for service_uuid, service in project_needed_services.items()
     ]
-    await logged_gather(*start_service_tasks, reraise=True)
+
+    result = await logged_gather(*start_service_tasks, reraise=True)
+    log.debug("Services start result %s", result)
+    for entry in result:
+        if "status" not in entry:
```

Comment:
are you sure you are not looking for ``service_state`` instead of ``status``??

``result`` is a list of bodies from the responses to [director_pai.start_service](https://github.com/ITISFoundation/osparc-simcore/blob/fb9870f260d9cfe7abed3db92b4d5014a127fe40/services/web/server/src/simcore_service_webserver/director/director_api.py#L73) and according to the OAS it returns a body with [these paremeters](
https://github.com/ITISFoundation/osparc-simcore/blob/1825744d853d2f0456abcf483e34676261608b24/services/director/src/simcore_service_director/api/v0/openapi.yaml#L1108)

Reply:
I am quite certain about this because I've actually checked with logs that this was the case. I will post back what it prints.

Reply:
```
Services start result [{'published_port': None, 'entry_point': '', 'service_uuid': 'fac8ba50-3243-4f30-ab3f-9b3e51ae49e8', 'service_key': 'simcore/services/dynamic/jupyter-octave-python-math', 'service_version': '1.4.1', 'service_host': 'jupyter-octave-python-math_fac8ba50-3243-4f30-ab3f-9b3e51ae49e8', 'service_port': 8888, 'service_basepath': '/x/fac8ba50-3243-4f30-ab3f-9b3e51ae49e8', 'service_state': 'pending', 'service_message': ''}]
```

I cannot  recall why this was added.

Reply:
If the `status` appears in the response body, it means that the API had failed. It's an odd behaviour. This is the moment where you can understand if a service did not start for some odd reason.